README.md

```diff
@@ -113,8 +113,27 @@ Key Transparency and its [Trillian](https://github.com/google/trillian) backend
 use a [MySQL database](https://github.com/google/trillian/blob/master/README.md#mysql-setup),
 which must be setup in order for the Key Transparency tests to work.
 
-Support
-------
+
+### Directory structure
+
+The directory structure of Key Transparency is as follows:
+
+* [**core**](core): main library source code. Core libraries do not import [impl](impl).
+    * [**api**](core/api): gRPC API definitions
+    * [**keyserver**](core/keyserver): keyserver implementation
+    * [**sequencer**](core/sequencer): sequencer implementation
+    * [**adminserver**](core/adminserver): admin server implementation
+* [**impl**](impl): environment specific modules:
+    * [**authentication**](impl/authentication): grpc authentication interceptor
+    * [**sql**](impl/sql): mysql implementations of storage modules
+    * [**integration**](impl/integration): environment specific integration tests
+* [**cmd**](cmd): binaries
+    * [**keytransparency-client**](cmd/keytransparency-client): Key Transparency CLI client
+* [**scripts**](scripts): scripts
+    * [**deploy**](scripts/deploy.sh): Deploy to Google Compute Engine
```

Comment:
There's also a top-level deploy dir - how does that differ from scripts/deploy?  It looks like scripts/deploy is actually scripts/deploy.sh, so should probably be *displayed* as such - currently it looks like deploy is a subdir of scripts.

Reply:
Good catch. This was incorrect.